contrib/debian/changelog.in

```diff
@@ -1,4 +1,10 @@
-openzfs-linux (2.2.0-0) unstable; urgency=low
+openzfs-linux (@VERSION@-1) unstable; urgency=low
+
+  * OpenZFS @VERSION@ is tagged.
+
+ -- Umer Saleem <usaleem@ixsystems.com>  Wed, 15 Nov 2023 15:00:00 +0500
+
+openzfs-linux (2.2.99-1) unstable; urgency=low
```

Comment:
Here, instead of `(2.2.99-1)`, it should be `(2.2.0-0)`. There is a warning because of this in the build:
`warning: the current version (2.2.1-1) is earlier than the previous one (2.2.99-1)`

Reply:
Thanks, there was a merge conflict when I cherry-picked it and I must have used the wrong line.  I'll fix it.